test-cases/tests/installation/a36-rhoam-operator-log-should-not-contain-errors-and-irrelevant.md

```diff
@@ -7,6 +7,7 @@ products:
       - 1.4.0
       - 1.7.0
       - 1.10.0
+      - 1.13.0
 estimate: 15m
 ---
 
```

Comment:
comment is not related to changes, just wanted to ask  - seems this test is for RHMI only (?) so maybe add it.

Reply:
this test case is actually just for RHOAM